### PR TITLE
Release 4.2.0

### PR DESCRIPTION
**Pull request by jankapunkt**
## Summary

This is an ongoing PR for the release `4.2.0` which will only include fixes and minor features (non-breaking).

## Linked issue(s)

See changelog.md section 4.2.0

#149

## Involved parts of the project

See changelog.md section 4.2.0

## Added tests?

See changelog.md section 4.2.0

## OAuth2 standard

See changelog.md section 4.2.0

## Reproduction

-

### CODE REVIEW

**Comment by jankapunkt** [comment]
If there are no further PRs to merge into 4.2.0 I will resolve the conflicts and create a new release. Is everybody okay with that? I can also create an rc-release if you first want to test it with your setups.

**Comment by bmxpiku** [comment]
I am awaiting this :)

**Comment by Uzlopak** [comment]
Just do it

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,38 @@
 ## Changelog
 
-## 4.1.1
```
Comment (HappyZombies) [inline CHANGELOG.md:3]: Are we removing the changelog for 4.1.1?

**Comment by jankapunkt** [comment]
:tada: all conflicts resolved, we should be good to go. One needs to approve. I will create a new GitHub release and publish to NPM

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@node-oauth/oauth2-server",
   "description": "Complete, framework-agnostic, compliant and well tested module for implementing an OAuth2 Server in node.js",
-  "version": "4.1.1",
+  "version": "4.1.0",
```
Comment (HappyZombies) [inline package.json:4]: Version downgraded by accident here.

**Review by HappyZombies — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (jankapunkt) [reply package.json:4]: I will set 4.2.0, thank you

**Review by HappyZombies — APPROVED** [review]
Code Review Completed

**Pull request merged**